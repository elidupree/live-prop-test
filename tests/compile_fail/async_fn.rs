use live_prop_test::live_prop_test;

#[live_prop_test(test_five)]
async fn five() -> i32 {
  5
}

fn test_five() -> impl FnOnce(&i32) {
  move |result| {
    assert_eq!(*result, 5);
  }
}

fn main() {}
