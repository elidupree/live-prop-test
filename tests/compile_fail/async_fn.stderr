error: live-prop-test doesn't support testing async fn items
 --> $DIR/async_fn.rs:4:1
  |
4 | async fn five() -> i32 {
  | ^^^^^
